pkg/evaluator/scope.go

```diff
@@ -23,6 +23,16 @@ func (s *scope) get(name string) (Value, bool) {
 	return s.outer.get(name)
 }
 
+func (s *scope) getScope(name string) (*scope, bool) {
+	if s == nil {
+		return nil, false
+	}
+	if _, ok := s.values[name]; ok {
+		return s, ok
```

Comment:
nit: I would make this `return s, true`. Since `ok` is always true in this branch, the literal makes it easier to read and understand at a glance.